ValveResourceFormat/Resource/Blocks/ResourceEditInfoStructs/SpecialDependency.cs

```diff
@@ -0,0 +1,31 @@
+using System.IO;
+using System.Text;
+using ValveResourceFormat.Serialization;
+using ValveResourceFormat.Serialization.KeyValues;
+
+namespace ValveResourceFormat.Blocks.ResourceEditInfoStructs
+{
+    public class SpecialDependency
+    {
+        public string String { get; set; }
+        public string CompilerIdentifier { get; set; }
+        public long Fingerprint { get; set; } // why long?
```

Comment:
Probably because GetIntegerProperty returns a long

Reply:
I will make it a uint regardless, and add a throw in KVObject constructor if value exceeds uint max.